core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -79,14 +79,11 @@ def start_navigator(self) -> None:
         # does not accept TCP master endpoints
         # self.start_mavlink_manager(Endpoint(local_endpoint))
 
-        # Check if subprocess is running and wait until it finishes
-        # Necessary since we don't have mavlink_manager running for navigator yet
```

Comment:
minor detail, you don't need to fix it, but these comments should have come out in the previous commit :)